pkg/controller/workload/workload_processor.go

```diff
@@ -587,10 +603,46 @@ func (p *Processor) handleAddressTypeResponse(rsp *service_discovery_v3.DeltaDis
 	}
 
 	_ = p.handleRemovedAddresses(rsp.RemovedResources)
+	p.compareWorkloadAndServiceWithHashName()
 
 	return err
 }
 
+// When processing the workload's response for the first time,
+// fetch the data from the /mnt/workload_hash_name.yaml file
+// and compare it with the data in the cache.
+func (p *Processor) compareWorkloadAndServiceWithHashName() {
+	var (
+		bk = bpf.BackendKey{}
+		bv = bpf.BackendValue{}
+	)
+
+	if kmeshbpf.GetStartType() == kmeshbpf.Normal {
+		return
+	}
+
+	log.Infof("reload workload config from last start")
+	kmeshbpf.SetStartType(kmeshbpf.Normal)
+	for str, num := range p.hashName.strToNum {
+		if p.WorkloadCache.GetWorkloadByUid(str) == nil && p.ServiceCache.GetService(str) == nil {
+			log.Debugf("GetWorkloadByUid and GetService nil:%v", str)
```

Comment:
We cannot assume all the strs are service or pod, can we?

Reply:
yes, I was careless here. I should add a check process in bpf_map.